packages/locales/lib/human/de.json

```diff
@@ -763,5 +763,25 @@
   "enter_translation": "Übersetzung eingeben",
   "individual_filters": "individuell gefiltert",
   "best_buddy": "Bester Kumpel",
-  "data_limit_reached": "Du hast kürzlich zu viele Daten angefordert und befindest dich im Cooldown bis {{until}}"
+  "data_limit_reached": "Du hast kürzlich zu viele Daten angefordert und befindest dich im Cooldown bis {{until}}",
+  "unknown_station": "unbekannte Kraftquelle",
+  "exclude_battle": "Dyna-Raids ausschließen",
+  "station": "Kraftquelle",
+  "stations": "Dynamax",
```

Comment:
Singular = Kraftquelle
plural = Dynamax ?
I think this should be "Kraftquellen"